cmd/jobsShow.go

```diff
@@ -118,8 +118,14 @@ func PrintJobTransfers(listTransfersResponse common.ListJobTransfersResponse) {
 		var sb strings.Builder
 		sb.WriteString("----------- Transfers for JobId " + listTransfersResponse.JobID.String() + " -----------\n")
 		for index := 0; index < len(listTransfersResponse.Details); index++ {
-			sb.WriteString("transfer--> source: " + listTransfersResponse.Details[index].Src + " destination: " +
-				listTransfersResponse.Details[index].Dst + " status " + listTransfersResponse.Details[index].TransferStatus.String() + "\n")
+			folderChar := ""
+			if listTransfersResponse.Details[index].IsFolderProperties {
+				folderChar = "/"
+				// TODO: reviewers, is it OK that this is hard coded to fwd slash (even for windows local sources?)
```

Comment:
>// TODO: reviewers, is it OK that this is hard coded to fwd slash (even for windows local sources?) [](start = 4, length = 99)

idea: We have a function to detect the resource's path separator. Consider using it instead

Reply:
Turns out there's not necessarily a path separator in every result string. Specifically, those at root level don't have one.  Given that fact, and the fact that `list` will mostly be used on cloud-based sources not local dirs, I'm going to leave this code as is.